addons/nametags/XEH_postInit.sqf

```diff
@@ -18,20 +17,15 @@ GVAR(showNamesTime) = -10;
     if (call FUNC(canShow)) then{ call FUNC(doShow); };
     // Return false so it doesn't block other actions
     false
-},
-{false},
-[29, [false, false, false]], false] call CBA_fnc_addKeybind; //LeftControl Key
+}, {false}, [29, [false, false, false]], false] call CBA_fnc_addKeybind; //LeftControl Key
 
-// Wait until the colors are defined before starting to draw the nametags
-["SettingsInitialized", {
-    // Draw handle
-    call FUNC(updateSettings);
-}] call EFUNC(common,addEventHandler);
+call FUNC(updateSettings);
 
 // Change settings accordingly when they are changed
-["SettingChanged", {
-    params ["_name"];
-    if (_name == QGVAR(showPlayerNames)) then {
+["CBA_SettingChanged", {
+    params ["_setting"];
+
+    if (_setting == QGVAR(showNames)) then {
         call FUNC(updateSettings);
     };
-}] call EFUNC(common,addEventHandler);
+}] call CBA_fnc_addEventHandler;
```

Comment:
It may be better to keep the ACE event system for the moment (maybe add a listener for "CBA_SettingChanged" that throw  a "SettingChanged"?)